roles/nuage_master/tasks/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Install Nuage Kubemon
+  sudo: true
+  yum: name={{ nuage_kubemon_rpm }} state=present
+
+- name: set kubeconfig 
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^kubeconfig line='kubeconfig{{':'}} {{ admin_config }}'
```

Comment:
Ideally, we should use a specially created kubeconfig for this purpose. When we get a bit closer, I can work on getting you the commands needed to create a service account and add the appropriate permissions based on the api objects that need to be interacted with.